CMake/fletch-tarballs.cmake

```diff
@@ -324,7 +330,11 @@ if (fletch_ENABLE_OpenCV OR fletch_ENABLE_ALL_PACKAGES OR AUTO_ENABLE_CAFFE_DEPE
 
   # Paired contrib repo information
   if (OpenCV_version VERSION_EQUAL 4.5.1)
-    set(OpenCV_md5 "cc13d83c3bf989b0487bb3798375ee08")
+    if (fletch_USE_PYPI_OPENCV)
+      set(OpenCV_md5 "0e178bd601b25a0a1ee0cd1e8c81bec0")
+    else()
+      set(OpenCV_md5 "cc13d83c3bf989b0487bb3798375ee08")
```

Comment:
I'll also note that we probably should be using something like sha256 or sha512 instead of md5 for content validation. Probably out of scope for this, but we should think about making that change.

Reply:
Yes, I would say out of scope for this PR.  We used md5 everywhere in Fletch.  So we should not just change it here.